libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/CustomManifestsForm.tsx

```diff
@@ -36,10 +36,12 @@ const AutosaveWithParentUpdate = ({
   const isEmpty = React.useMemo<boolean>(() => {
     return isEqual(values, emptyValues);
   }, [values, emptyValues]);
-  const isAutoSaveRunning = useFormikAutoSave();
+  const isAutoSaveRunning = useFormikAutoSave(1000);
```

Comment:
magic numbers...

Reply:
I think with the other changes I don't need this timeout. I'll check it later